ethclient/ethclient.go

```diff
@@ -51,6 +55,10 @@ func NewClient(c *rpc.Client) *Client {
 	return &Client{c}
 }
 
+func (ec *Client) Close() {
+	ec.Close()
```

Comment:
This is an infinite loop ;)

Reply:
You are obviously right, and it's a typo :). Fixed.